### PR TITLE
better ai

### PR DESCRIPTION
**Pull request by hkubus**

### CODE REVIEW

**Review by Nyameliaaaa — DISMISSED** [review]
im not straight but ok openai :trollface:

File: src/context/ai.ts [hunk 1]
```diff
@@ -1,27 +1,41 @@
...
     }
-    if (client.ai.queue.some(q => q.object.user.id === interaction.user.id)) {
-        interaction.reply({
-            content: client.i18n.t('AI_LIMIT', { lng: interaction.locale }),
-            ephemeral: true
+    if (client.aiCooldown.has(interaction.user.id)) {
+        return interaction.reply({
+            content: client.i18n.t('AI_COOLDOWN', { lng: interaction.locale })
         });
     }
```
Comment (SuperchupuDev) [inline src/context/ai.ts:19]:
```suggestion
    }
    await interaction.deferReply();
```

File: src/context/ai.ts [hunk 2]
```diff
@@ -1,27 +1,41 @@
...
+        body: JSON.stringify({
+            model: 'ada',
+            question: content,
+            temperature: 0.8
+        })
+    }).then(res => res.json())) as any;
+    const embed = new Embed(client, interaction.locale)
+        .setLocaleTitle('AI_TITLE')
+        .setLocaleDescription('AI_DESCRIPTION', { text: response.answers[0] });
+    interaction.reply({ embeds: [embed] });
```
Comment (SuperchupuDev) [inline src/context/ai.ts]:
```suggestion
    interaction.editReply({ embeds: [embed] });
```

File: src/commands/ai.ts [hunk 3]
```diff
@@ -2,23 +2,47 @@ import FluorineClient from '@classes/Client';
...
-    if (client.ai.queue.some(q => q.object.user.id === interaction.user.id)) {
-        interaction.reply({
-            content: client.i18n.t('AI_LIMIT', { lng: interaction.locale }),
-            ephemeral: true
+    if (client.aiCooldown.has(interaction.user.id)) {
+        return interaction.reply({
+            content: client.i18n.t('AI_COOLDOWN', { lng: interaction.locale })
         });
     }
     interaction.deferReply();
```
Comment (SuperchupuDev) [inline src/commands/ai.ts]:
```suggestion
    await interaction.deferReply();
```

**Review by SuperchupuDev — CHANGES_REQUESTED** [review]

**Review by SuperchupuDev — APPROVED** [review]

**Review by Nyameliaaaa — APPROVED** [review]
sex

**Pull request merged**